### PR TITLE
add version from Docker image via environment

### PR DESCRIPTION
**Pull request by stephankn**
improves  #404 ,

call 

    docker build --build-arg GIT_VERSION=`git describe --dirty`

to pass the current version as environment to the image.

### CODE REVIEW

File: osmose_run.py [hunk 1]
```diff
@@ -65,13 +65,13 @@ class analyser_config:
   pass
 
 def get_version():
-    cmd  = ["git", "describe"]
+    cmd  = ["git", "describe", "--dirty"]
```
Comment (frodrigo) [inline osmose_run.py]: Remove this change from this PR, please.

File: osmose_run.py [hunk 2]
```diff
@@ -65,13 +65,13 @@ class analyser_config:
...
     try:
         proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
         version = proc.stdout.readlines()[0].strip()
     except:
         version = "(unknown)"
-    return version
+    return os.getenv('OSMOSE_VERSION', version)
```
Comment (frodrigo) [inline osmose_run.py]: If there is `OSMOSE_VERSION` do not call to git before.

File: docker/Dockerfile [hunk 3]
```diff
@@ -34,6 +34,8 @@ ENV PYTHONPATH /opt/osmose-backend
 
 ADD . /opt/osmose-backend/
 
+ARG GIT_VERSION
```
Comment (frodrigo) [inline docker/Dockerfile]: Can we make the `git describe` in Dockerfile?

Comment (stephankn) [reply docker/Dockerfile]: This is not possible, as the .git file is already excluded inside the Docker build environment

**Comment by stephankn** [comment]
do you need further changes to have this merged?

**Comment by frodrigo** [comment]
It's OK. Thank you.

**Pull request merged**